### DISCUSSION TITLE
Navigation category headings

### DISCUSSION THREAD

**Discussion by djgrant** [retypeapp/retype#332] [Ideas]
I would like to add category headings to the left side navigation, similar to gitbook, docusaurous etc.

The collapsible folder pattern is really nice in some instances. In others, it is more appropriate to break up a long list of top-level links with category headings.

Example of both patterns being used:

<img width="296" alt="Screenshot 2022-05-16 at 19 24 07" src="https://user-images.githubusercontent.com/1670902/168657974-d724dd45-910e-4939-a0f5-d3bb5ab220ff.png">

**Comment by rgbinnovation** [comment]
I came here to suggest this exact thing. I find it strange that this is not already available?
